### PR TITLE
PKG-1112: Fix miniconda rst scripts

### PR DESCRIPTION
**Pull request by pseudoyim**
Jira ticket: https://anaconda.atlassian.net/browse/PKG-1112

- `docs/create_miniconda_rst.py`
  - (FIXED) The “Latest” line at the top was not be getting updated and had to be manually done. Values for the global vars should be updated in the script _before_ running it.
  - (FIXED) The hash for the `latest` `win-32` Miniconda was getting deleted. Should remain "frozen" as: `Python 3.9 Miniconda3 Windows 32-bit 67.8 MiB 4fb64e6c9c28b88beab16994bfba4829110ea3145baa60bda5344174ab65d462` because we are no longer building `win-32` installers.
  - (FIXED) All of the `py` version links (not just for `latest`) need to get updated to point to the new installer versions. See [this comment](https://github.com/conda/conda-docs/pull/838#discussion_r1101137571) from DBast.

- `docs/create_miniconda_hash_rst.py`
  - (DONE) Order the rows in a way that makes intuitive sense (e.g. py310, py39, py38…).
  - (DONE) Ensure no more “malformed table” errors (increased char width for table columns).

- (DONE) Add steps to README for local builds to preview changes.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
 
 # General information about the project.
 project = u'conda'
-copyright = u'2017, Anaconda, Inc.'
+copyright = u'2023, Anaconda, Inc.'
```
Comment (kathatherine) [inline docs/source/conf.py:58]: The copyright doesn't need to be updated here. That's when the docs were created and it's not that useful (and probably not even correct, legally speaking) to update that every year.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Thanks for this! Just a few things

File: docs/source/conf.py (hunk 1 above)
Comment (pseudoyim) [reply docs/source/conf.py]: Changed it back. Thanks!

File: README.md [hunk 2]
```diff
@@ -1,11 +1,47 @@
...
+
+## Local builds
+
+You can build this docs repo on your local machine to preview changes in your browser.
+
+NOTE: These steps have only been tested on MacOS
+
+### Setup and activate a `sphinx` environment
+
+1. If you haven't already, create a conda environment with `sphinx` installed.
```
[the remark this answers is not in the document]
Comment (pseudoyim) [reply README.md:20]: Markdown lets you write numbered lists as all `1.`s (so you don't have to change them all when you insert a new number in the middle somewhere). It renders correctly when viewed:
Screen Shot 2023-02-23 at 5 10 24 PM

[the remark this answers is not in the document]
Comment (kathatherine) [reply README.md:20]: Oh, gotcha! TIL

**Review by kathatherine — APPROVED** [review]
Thanks so much for this, Paul!

**Pull request merged**